java/gradle.java/test/unit/data/javasimple/src/main/java/test/App.java

```diff
@@ -0,0 +1,18 @@
+package test;
+
+import java.util.Collections;
+
+public class App {
+    public static void main(String[] args) {
+        for (int i = 0; i < args.length; i++) {
+            System.err.println("args." + i + "=" + args[i]);
+        }
+        String[] props = Collections.list(System.getProperties().propertyNames()).stream().
+            filter(n -> n.toString().startsWith("test.")).
```

Comment:
Are you sure the JDK doesn't (and will not) use any property prefixed with `test.`?

Reply:
Prefix changed in 5d055fdd54